integration-tests/jiva/volume/provision_test.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package volume
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	pvc "github.com/openebs/maya/pkg/kubernetes/persistentvolumeclaim/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	replicaLabel = "openebs.io/replica=jiva-replica"
+	ctrlLabel    = "openebs.io/controller=jiva-controller"
+	accessModes  = []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce}
+	capacity     = "5G"
+	pvcObj       *corev1.PersistentVolumeClaim
+	err          error
+)
+
+var _ = Describe("[jiva] TEST VOLUME PROVISIONING", func() {
+	var (
+		pvcName = "jiva-volume-claim"
+	)
+
+	BeforeEach(func() {
+
+	})
+
+	When("jiva pvc with replicacount 1 is created", func() {
+		It("should create 1 controller pod and 1 replica pod", func() {
+
+			By("building a persistent volume claim")
+			pvcObj, err = pvc.NewBuilder().
+				WithName(pvcName).
+				WithNamespace(nsName).
+				WithStorageClass(scName).
+				WithAccessModes(accessModes).
+				WithCapacity(capacity).Build()
+			Expect(err).ShouldNot(
+				HaveOccurred(),
+				"while building persistentvolumeclaim {%s} in namespace {%s}", pvcName, nsName)
+
+			By("creating a jiva pvc")
```

Comment:
Why do we change the message style?
Check for both the `By()` messages. First one talks about building & next one should talk about creating it. However, by changing the messaging style, we are making them different.

```go
By("building a persistent volume claim")
// ...

By("creating this persistent volume claim")
// ...
```